ddp-workspace/projects/ddp-testboston/src/app/app-routes.ts

```diff
@@ -9,5 +11,6 @@ export const AppRoutes = {
     CovidSurvey: 'covid-survey',
     Dashboard: 'dashboard',
     Password: 'password',
-    SessionExpired: 'session-expired'
+    SessionExpired: 'session-expired',
+    UserRegistrationPrequal: 'user-registration-prequal'
```

Comment:
The path for the component.I believe that we want to have component live in home page. Doable but punting on that for now just to make it easier to work with and get to, I give its own route.

Reply:
As I remember, Jen asked to use `join` route name for this page.

Reply:
For the reference https://broadinstitute.slack.com/archives/G012Y2S45KN/p1589484057371700

Reply:
385e0df